app/docs/md/conventions/components.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Components
+---
+
+Components are another option for reusable building blocks of your Enhance application. They are single file components wrapping your HTML, CSS and JavaScript in a portable web component. Components live in the `app/components/` folder in Enhance projects.
+
+## Naming
+
+The file name of your component will be the tag name you author with. Meaning `app/components/my-card.mjs` will be authored as `<my-card></my-card>` in your HTML page. Enhance components are HTML custom elements, so they [require two or more words separated by a dash](/docs/elements).
```

Comment:
This is pedantic so feel free to ignore, but I might flip this first sentence so the causation is more logical: 

```suggestion
The tag name of your component will be mapped to its file name. Meaning `app/components/my-card.mjs` will be authored as `<my-card></my-card>` in your HTML page. Enhance components are HTML custom elements, so they [require two or more words separated by a dash](/docs/elements).
```

Reply:
Grammarly, actually complains about the updated suggestion.

Reply:
Oh yes, I see, good call. In that case, I'd suggest 'The tag name of your component is determined by its file name.'